packages/config/src/chainConfig/types.ts

```diff
@@ -49,11 +49,18 @@ export type ChainConfig = {
   GLOAS_FORK_EPOCH: number;
 
   // Time parameters
-  SECONDS_PER_SLOT: number;
+  SECONDS_PER_SLOT: number; // Deprecated
+  SLOT_DURATION_MS: number;
   SECONDS_PER_ETH1_BLOCK: number;
   MIN_VALIDATOR_WITHDRAWABILITY_DELAY: number;
   SHARD_COMMITTEE_PERIOD: number;
   ETH1_FOLLOW_DISTANCE: number;
+  PROPOSER_REORG_CUTOFF_BPS: number;
+  ATTESTATION_DUE_BPS: number;
+  AGGREGRATE_DUE_BPS: number;
```

Comment:
There's a typo here, that's already being fixed in https://github.com/ethereum/consensus-specs/pull/4524 but it would be good if we wait for spec release and update our config tests to make sure all constants match with the spec

```suggestion
  AGGREGATE_DUE_BPS: number;
```

Reply:
🤦 can't believe I didn't catch this either